.github/workflows/release_workflow.yml

```diff
@@ -40,7 +40,14 @@ jobs:
       - name: Build Conda Package
         run: conda build -c conda-forge --output-folder . .
 
-      - name: Publish to Anaconda
+      - name: Publish to Anaconda (e3sm channel)
```

Comment:
@xylar Testing this on my fork before I merge. It was a quick addition, so let me know if the bash statement can be improved (I'm not a bash expert).

Reply:
Looks fine to me (but bash isn't my favorite either).